docs/application/web/guides/multimedia/metadata.md

```diff
@@ -44,6 +63,105 @@ When all the needed metadata is extracted, the file handle needs to be released
    fileHandle.release();
    ```
 
+## Extract artwork
+
+1. Create a file handle using the `createFileHandle()` method. Your file needs to have an artwork included.
+
+   ```javascript
+   var filePath = "videos/sample.mp3";
+   var fileHandle = tizen.metadata.createFileHandle(filePath);
+   ```
+
+2. When the file handle is properly created, artwork can be extracted:
+
+   ```javascript
+   var artwork = fileHandle.getArtwork();
+   ```
+
+3. *Blob* object can be easily shown html code in your application.
```

Comment:
Can you rephrase this sentence? It doesn't sound clear. (It occurs 3 times in this file)

Reply:
```suggestion
3. *Blob* object can be easily shown in your application in html *image* tag.
```
will this be ok?

Reply:
Yes, but remember that '<' has to be escaped in md - by '\<' or '&lt;'.